packages/builder/src/mrl/MrlBuilder.interfaces.ts

```diff
@@ -0,0 +1,33 @@
+import type {
+  AnyChain,
+  ChainAsset,
+  EvmParachain,
+} from '@moonbeam-network/xcm-types';
+import type { ApiPromise } from '@polkadot/api';
+import type { HexString } from '@polkadot/util/types';
+import type { BuilderPrams, ConfigBuilder } from '../builder.interfaces';
+import type { WormholeConfig } from './wormhole/wormhole';
+import type { ExtrinsicConfig } from '../extrinsic';
+import type { ContractConfig } from '../contract';
+
+export type MrlConfigBuilder = ConfigBuilder<
+  ContractConfig | ExtrinsicConfig | WormholeConfig,
+  MrlBuilderParams
+>;
+
+export interface MrlBuilderParams extends BuilderPrams<AnyChain> {
+  isAutomatic: boolean;
```

Comment:
isn't this in `MrlConfig` already? is it needed here as well?

Reply:
We need it here because when we build transfer config we need to pass it. But in general, I'm still not sure where to put it. Because we need to be able to receive it in the later stages, because it needs to be switchable in the UI.